spec/Binary/20-BINARY_FORMAT.md

```diff
@@ -1,3 +1,54 @@
 # Binary format
 
-TBD
\ No newline at end of file
+Binary format document describes how to encode each field - `traceparent` and
+`tracestate`.
+
+# `Traceparent` binary format
+
+The field `traceparent` encodes the version of the protocol and fields
+`trace-id`, `span-id` and `recorded`. Each field starts with the one byte field
+identifier with the field value following immediately after it. Field
+identifiers are used as markers for additional verification of the value
+consistency and may be used in future for the versioning of the `traceparent`
+field.
+
+``` abnf
+traceparent     = version "0" trace-id "1" parent-id "2" recorded
```

Comment:
The `version` field should probably be defined above just to make it clear that it's one byte. I suggest updating this to be similar to how it's defined in the HTTP header spec ([2.2.2 Field value](https://w3c.github.io/trace-context/#field-value)), where there is a `value` field specifying the entire binary format consisting of a `version` and a `version-format` and then the `version-format` is defined on its own line.

Reply:
makes sense. Good idea